R/matomo_package_query.R

```diff
@@ -18,13 +19,13 @@
 #'  method = "API.getMatomoVersion"
 #' )
 #'
-#' # Create the url
-#' url <- httr::modify_url(matomo_hostname(), query = query)
-#'
 #' # Make the query
-#' matomo_package_query(url)
+#' matomo_package_query(query = query)
 matomo_package_query <-
-  function(url = NULL, query = NULL) {
+  function(query = NULL, url = NULL) {
```

Comment:
making query the first argument which makes it easier since often that will be the only needed argument.

now of course this is acceptable only if not many people relied on this exported function. I assume the package was fairly confidential and that therefore not much code will be broken, if any?